cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkLoginHandler.java

```diff
@@ -88,6 +91,14 @@ public void setInstitutionLogin(final Boolean institutionLogin) {
             this.institutionLogin = institutionLogin;
         }
 
+        public boolean isOrcidRedirect() {
```

Comment:
Boolean properties accessors uses the **is** instead of **get**. Consider doing the same for `institutionLogin`.